### ISSUE TITLE
Travis CI: new job created on a GitHub PR if a build is restarted

### ISSUE THREAD

**Issue by vstinner** [python/core-workflow#374]
In short, I need a GitHub admin to remove an outdated Travis CI webhook: https://travis-ci.community/t/duplicate-job-attempts-for-pr-builds-causing-queued-build-created-problems/9776/2

who are the admins of the cpython GitHub project? cc @ewdurbin @pablogsal @Mariatta @ned-deily 

Long story: on my PR https://github.com/python/cpython/pull/22143 the "doc" job of Travis CI was stuck in "queued", whereas the job completed (successfully!). Since Travis CI is mandatoary, I cannot merge my PR. So I clicked on (Restart build) at Travis CI. Problem: the old job is still around, and a new job was added. Even if the new job completed successfully, the old job is still stuck and so I still cannot merge my PR.

Closing/Reopening the PR doesn't work around the problem: the old job is still there and still stuck.

It's the https://travis-ci.community/t/duplicate-job-attempts-for-pr-builds-causing-queued-build-created-problems/9776 bug. The solution is to remove an old webhook from the https://github.com/python/cpython/ project (admin page).

**Comment by vstinner** [comment]
Note: this issue is a side effect of the fix of the previous Travis CI issue, https://github.com/python/core-workflow/issues/371

**Comment by ewdurbin** [comment]
web hook has been removed.

**Comment by vstinner** [comment]
> web hook has been removed.

Thanks @ewdurbin! To check that the issue is really fixed, I clicked on (Restart build) on https://travis-ci.com/github/python/cpython/builds/183369816 of my https://github.com/python/cpython/pull/22150 PR. The PR only has the new build, the old one is gone. It works as expected. Good!

I checked a personal project that I recently migrated from travis-ci.org to travis-ci.com, and it also had the remaining webhook. I also removed it.
